src/client/client_utils.cpp

```diff
@@ -97,4 +79,39 @@ std::string clientEncrypt(const std::string &value) {
         (size_t)engine.encryptNonDeterministic(value, cipher_text.get());
     std::string updated_val((const char *)cipher_text.get(), out_len);
     return updated_val;
+}
+
+void parseArgs(int argc, char *argv[], ClientConfig &config) {
+    argparse::ArgumentParser program("ortoa-tee");
+
+    program.add_argument("--seed").default_value(std::string{""});
+
+    program.add_argument("--nthreads").default_value(16).scan<'d', int>();
```

Comment:
I know why this flag isn't working. It's because of the script in `scripts/ortoa-lib.sh` -- looks like I added some undesired behaviour.

```sh
while getopts ":h" option; do
    case "${option}" in
        h) echo "${HELP}"; return 0 ;;
    esac
done
```

Remove this piece of the script and then it will probably work as expected.

Reply:
Removed this and works as expected